frontend-app/src/yash_ui_task/Screen.jsx

```diff
@@ -0,0 +1,28 @@
+import '../App.css';
+import React, { useState } from 'react';
+import 'antd/dist/antd.css';
+import {Divider,PageHeader, Typography} from "antd";
+import Student from './Components/student'; 
+import Teacher from './Components/teacher';
+import Layout from 'antd/lib/layout/layout';
+import { Header } from 'antd/lib/layout/layout';
+
+export default function ScreenUi()
+{
+    return (
+        <div>
+        <Layout>
+          <Header><Typography>Record</Typography></Header>
```

Comment:
Indentation and use <Title> present in <Typography>

Reply:
done